ZenPacks/zenoss/Dashboard/browser/resources/js/defaultportlets.js

```diff
@@ -627,6 +627,24 @@
             });
             this.callParent(arguments);
         },
+        validateSiteUrl: function(siteUrl) {
+            // check if given url is valid
+            var url_pattern = /(http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/;
```

Comment:
Please, use `camelCase` for the variable name according to Ext JS naming convention